<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QComboBox" name="account_list">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>현재 계좌</string>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>이름</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_con">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>390</y>
     <width>521</width>
     <height>181</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>종목코드</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>종목이름</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>현재가</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>전일종가</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>기타</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="cbox_con">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>410</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_maedo">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>521</width>
     <height>241</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>종목코드</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>종목이름</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>현재가</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>매입금액</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>평가금액</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>조건검색 목록</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>보유 주식 목록</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>390</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>조건검색 선택</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sell_btn">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>310</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>선택 종목 매도</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>선택 종목 : </string>
   </property>
  </widget>
  <widget class="QLabel" name="view_selec_coin_lbl">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AmeriGarmnd BT</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_jango">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>181</width>
     <height>221</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>예수금</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>D+2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>출금가능금액</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>주식매수총액</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>평가금액합계</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>금액</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="buy_amount_edit">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>138</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>100,000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>138</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>매수금액:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>164</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>매도설정:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="order_amount_4">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>164</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>164</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="auto_mm_start">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>210</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;

background-color:red;
</string>
   </property>
   <property name="text">
    <string>자동매매 시작</string>
   </property>
  </widget>
  <widget class="QPushButton" name="auto_mm_end">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>250</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;

background-color:blue;
</string>
   </property>
   <property name="text">
    <string>자동매매 중단</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>881</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;

background-color:blue;
</string>
   </property>
   <property name="text">
    <string>WIN-STOCK</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>110</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>15</hour>
     <minute>15</minute>
     <second>0</second>
     <year>2000</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>110</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>매수시간:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="real_log_widget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>181</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>[ 실시간 기록 ]</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
